ui/desktop/tests/integration/helpers/order-by-test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, findAll } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Helper | order-by', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    this.set('inputValue', [
+      { key: 'username', value: 1 },
+      { key: 'password', value: 2 },
+      { key: 'none', value: 3 },
```

Comment:
What happens if you have other keys that are in between username and password? Does that order make sense?